spec/monasca_client_tests.js

```diff
@@ -0,0 +1,26 @@
+import nock from 'nock';
+
+export class monasca_api_mock {
```

Comment:
Is this left over from testing? It doesn't look like we use it? Can we delete it?